pages/docs/platform/monitor/insights.mdx

```diff
@@ -0,0 +1,209 @@
+import { Info } from "src/shared/Docs/mdx";
+
+export const description = 'Query and analyze event data with SQL in the Inngest platform.';
+
+# Insights
+
+Inngest Insights allows you to query and analyze your event data using SQL directly within the Inngest platform. Every event sent to Inngest contains valuable information, and Insights gives you the power to extract meaningful patterns and analytics from that data.
+
+<Info>
+  Insights support is currently in Public Beta. Some details including SQL syntax and feature availability are still subject to change during this period. Read more about the Public Beta [roadmap here](#roadmap).
```

Comment:
not sure if we should link to https://www.inngest.com/docs/release-phases as well